ConeSearch.tex

```diff
@@ -112,101 +114,122 @@ \section{Introduction}
-\begin{center}
+\begin{table}[th] \begin{center}
 \begin{tabular}{p{0.25\textwidth}p{0.25\textwidth}p{0.25\textwidth}}
-\sptablerule
-\textbf{resource type}&\textbf{resource name}&\textbf{required}\\
-\sptablerule
-\{query\} & service specific & yes\\
-VOSI-capabilities & /capabilities & should\\
-VOSI-availability & (/availability) & should\\
-\sptablerule
-\label{table:resources}
-\end{tabular}
-\caption{Simple Cone Search resources tree}
-\end{center}
+\sptablerule \textbf{resource type}&\textbf{resource
+name}&\textbf{required}\\ \sptablerule \{query\} & service specific &
+yes\\ VOSI-capabilities & /capabilities & should\\ VOSI-availability &
+(/availability) & should\\ \sptablerule \label{table:resources}
+\end{tabular} \caption{Simple Cone Search resources tree} \end{center}
 \end{table}
 
 It \textbf{must} have one \{query\} resource and \textbf{should} have a
 VOSI-capability resource and a VOSI-availability one. The
 VOSI-capability \textbf{must} be a sibling to the \{query\} one.
 
-\subsection{\{query\} resource}
-\label{sec:basepar}
-The Simple Cone Search \{query\} resource \textbf{must} meet the following requirements:
-\begin{itemize}
-	\item the resource \textbf{must} respond to requests submitted using
-an HTTP GET query string, and \textbf{should} respond to HTTP POST query
-actions (as described in later in this same section);
-	\item the response, by default, \textbf{must} be a valid VOTable
-          document, providing a table containing the sources of the catalogue
-          whose positions and/or timestamps are within the cone and/or time
-          interval described in the query request (see Sec.~\ref{sec:response});
-	\item in case of error, the response \textbf{must} follow
-          prescriptions as described in Sec.~\ref{sec:error}.
-\end{itemize}
-
-The Simple Cone Search service \texttt{\{query\}} resource is a synchronous web resource, as described by DALI, of the form
-$$\hbox{\nolinkurl{http://<server>/<local_path>?} }$$
-To this base URL the query parameters, described hereafter, are attached to build the query request to be submitted through HTTP.
-
-Usage of fixed, custom query parameters, defined by the service provider to build a base URL of the form
+\subsection{\{query\} resource} \label{sec:basepar} The Simple Cone
+Search \{query\} resource \textbf{must} meet the following requirements:
+\begin{itemize} \item the resource \textbf{must} respond to requests
+submitted using an HTTP GET query string, and \textbf{should} respond to
+HTTP POST query actions (as described in later in this same section);
+\item the response, by default, \textbf{must} be a valid VOTable
+document, providing a table containing the sources of the catalogue
+whose positions and/or timestamps are within the cone and/or time
+interval described in the query request (see Sec.~\ref{sec:response});
+\item in case of error, the response \textbf{must} follow prescriptions
+as described in Sec.~\ref{sec:error}.  \end{itemize}
```

Comment:
The old list layout was a bit easier to read. Just having the \begin and ]\end commands on their own line would help.

Reply:
fixed in commit 98611e2